keps/sig-contributor-experience/0005-community-site.md

```diff
@@ -97,14 +114,47 @@ Hopefully the content previously contained in [umbrella issues][] will be tracke
 
 ## Implementation History
 
-## Drawbacks [optional]
+## Drawbacks
+
+The biggest drawback is that this is yet another thing to keep running.
+Currently the markdown files are workable but not super discoverable.
+However, they utilize the familiar mechanisms and do not require extra effort or understanding to publish.
+
+The current mechanisms also scale across orgs and repos.
+This is a strength as the information is close to the code but also a big disadvantage as it ends up being much less discoverable.
+
+## Alternatives
+
+One alternative is to do nothing.
+However, the smattering of markdown through many repos is not scaling and is not discoverable via Google or for other members of the community.
+
+The main alternative here is to build something that is integrated into the user facing kubernetes.io site.
+This is not preferred for a variety of reasons.
+
+* **Workflow.** Currently there is quite a bit of process for getting things merged into the main site.
+  That process involves approval from someone on SIG-Docs from an editorial point of view along with approval for technical accuracy.
```

Comment:
We're working on streamlining this process and getting PRs through more quickly. Many PRs currently get merged without two sets of review; technical accuracy is at issue only for, well, highly technical topics.

Reply:
It's worth pointing out that an OWNERS file provides directory-level ownership (hypothetically, in kubernetes/website, docs/contrib/*) and can disallow permissions inheritance (`no_parent_owners: true`) to limit and isolate automation assignments. 

All of which is to say that it's possible to achieve review isolation on the main site.